### PR TITLE
 Feature/iops 1624 practitioner

### PR DESCRIPTION
**Pull request by digitalhealthmartin**

### CODE REVIEW

**Comment by digitalhealthmartin** [comment]
Forgot to add examples. Will do this next.

**Comment by digitalhealthmartin** [comment]
Example files added. Ready for review.

File: examples/England-Practitioner-BasicPractitioner-Example.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<Practitioner xmlns="http://hl7.org/fhir">
+    <id value="England-Practitioner-BasicPractitioner-Example" />
```
Comment (KazeemHamzat) [inline examples/England-Practitioner-BasicPractitioner-Example.xml:2]: Just a suggestion: England-Practitioner-PractitionerDetails-Example

File: examples/England-Practitioner-Validation-Example.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<Practitioner xmlns="http://hl7.org/fhir">
+    <id value="England-Practitioner-Validation-Example" />
```
Comment (KazeemHamzat) [inline examples/England-Practitioner-Validation-Example.xml:2]: England-Practitioner-PractitionerDetailsValidation-Example"

**Review by KazeemHamzat — COMMENTED** [review]
Some suggestions on examples ID that can better identify the examples

**Comment by digitalhealthmartin** [comment]
Removed unrelated files.

**Review by SBajanki1 — APPROVED** [review]

**Review by SBajanki1 — DISMISSED** [review]

**Comment by digitalhealthmartin** [comment]
Added CS

**Comment by digitalhealthmartin** [comment]
**OLD PR. DO NOT MERGE. CLOSING PR.**

**Pull request closed without merging**